.github/workflows/lint_test_build.yml

```diff
@@ -18,9 +18,30 @@ jobs:
       - uses: ./.github/workflows/rafiki/env-setup
       - run: pnpm checks
 
-  backend:
+  graphql:
     runs-on: ubuntu-latest
     needs: checkout
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/workflows/rafiki/env-setup
+
+      - name: generate graphql
+        run: pnpm --filter backend generate
```

Comment:
Should we add this check for the `auth` package's graphQL API as well?

Reply:
Oh yes. Forgot that it also has a graphql server now.